PyInstaller/utils/hooks/hookutils.py

```diff
@@ -29,7 +29,7 @@
 
 # These extensions represent Python executables and should therefore be
 # ignored when collecting data files.
-PY_IGNORE_EXTENSIONS = set(['.py', '.pyc', '.pyd', '.pyo', '.so', 'dylib'])
+PY_IGNORE_EXTENSIONS = set(['.py', '.pyc', '.pyd', '.pyo', '.so', '.dylib'])
```

Comment:
This is weird. While testing this pull request and keeping the original value `dylib` it works. I will keep investigating more.